qiita_db/test/test_analysis.py

```diff
@@ -549,13 +567,25 @@ def test_build_biom_tables(self):
         sql = """SELECT EXISTS(SELECT * FROM qiita.filepath
                  WHERE filepath_id=%s)"""
         obs = self.conn_handler.execute_fetchone(sql, (new_id,))[0]
-
         self.assertTrue(obs)
 
         sql = """SELECT * FROM qiita.analysis_filepath
                  WHERE analysis_id=%s ORDER BY filepath_id"""
         obs = self.conn_handler.execute_fetchall(sql, (self.analysis.id,))
-        exp = [[1L, 14L, 2L], [1L, 15L, None], [1L, new_id, None]]
+        exp = [[1L, 12L, 2L], [1L, 13L, None], [1L, new_id, 2L]]
+        self.assertEqual(obs, exp)
+
+    def test_build_biom_tables_duplicated_samples_merge(self):
```

Comment:
Yeah, looking at the tests it is clear that the parameter `duplicated_samples` causes different behavior in these two functions - can we normalize?